Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -13,6 +13,7 @@
 #endif // UNITY_WSA
 
 #if WINDOWS_UWP
+using Windows.Foundation.Metadata;
```

Comment:
Is this needed after the move to the MRTK API?

Reply:
good catch!